Orange/widgets/visualize/owvenndiagram.py

```diff
@@ -762,6 +764,14 @@ def expanded(feat):
         indices = inst_by_id[inst_id]
         instance = newtable[i]
 
+        for var in domain.variables + domain.metas:
+            if var in idvarlist:
+                continue
+            if numpy.isnan(instance[var]):
+                for ind in indices:
+                    if not numpy.isnan(table[ind, var]):
```

Comment:
Should be fixed in the updated version.

Expanded features were already handled below the new for loop, so they should be skipped here.